recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma.cfg

```diff
@@ -0,0 +1,31 @@
+seclabel='system_u:system_r:domU_t'
+
+# Guest name
+name = "DomA"
+
+# Kernel image to boot
+kernel = "/xt/doma/Image"
+
+device_tree = "/xt/domd/doma.dtb"
+
+coproc=[ "/soc/gsx@fd000000" ]
+irqs=[ 151 ]
+iomem=[ "0xfd010,1@0xfd000" ]
+
+# Kernel command line options
+extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/doma ip=192.168.1.12 rw rootwait console=hvc0 cma=256M"
```

Comment:
I doubt your Android will boot with this command line.

Reply:
This is by intention to be able to boot with Android kernel, but prod devel's domu rootfs